optout/src/test/java/gov/cms/ab2d/optout/OptOutProcessorTest.java

```diff
@@ -142,4 +152,27 @@ void getRecordStatusTest() {
         assertEquals(RecordStatus.REJECTED.toString(), optOutProcessing.getRecordStatus());
     }
 
+    @Test
+    void getOptOutResultsTest() throws SQLException {
+        final String OPT_IN_RESULTSET_STRING = "optin";
```

Comment:
I believe, sonarqube will complain on uppercase letters here. Usually uppercase letters should be used only for global (or class) variables